sphinx/texinputs/sphinx.sty

```diff
@@ -11,14 +11,17 @@
 \@ifclassloaded{memoir}{}{\RequirePackage{fancyhdr}}
 
 \RequirePackage{textcomp}
-\RequirePackage{fancybox}
+% fancybox was used in Sphinx <= 1.4.1 for \shadowbox and Sbox environment.
+% \RequirePackage{fancybox}
```

Comment:
To keep dependency carefully, could you keep loading the `fancybox` package? And give a comment as "fancybox will be removed at Sphinx-1.5" please.
It might break the documents if it use the package in their customization and extension. We should keep compatibility during same major version. Probably nobody depends it :-p

After that, please remove the `RequirePackage` definition in master branch with append that to "incompatible changes" section on CHANGES file.